packages/experimental/Icon/src/Icon.tsx

```diff
@@ -18,8 +18,12 @@ function renderRasterImage(iconProps: IconProps) {
 }
 
 function fontFamilyFromFontSrcFile(fontSrcFile: string, fontFamily: string): string {
-  const asset = assetRegistry.getAssetByID(fontSrcFile);
-  return `${fontFamily}#${asset.httpServerLocation}/${asset.name}.${asset.type}`;
+  if (Platform.OS == 'windows') {
```

Comment:
A comment would be nice in the two places we have this windows platform check to let folks know the plan for macOS/iOS font icon support and why it's not available

Reply:
Font icon support should be available; I've just been having trouble getting the font bundled in. That's why I added an Arial-based font icon to the test page.

Reply:
Can you comment that in code? Sometimes that's not always obvious to someone else looking at this once it gets checked in

Reply:
Done